src/interactive/azext_interactive/azclishell/recommendation.py

```diff
@@ -237,59 +243,82 @@ def get_recommend_from_api(command_list, type, top_num=5, error_info=None):  # p
     if 'data' in response.json():
         recommends = response.json()['data']
 
-    return recommends
+    if 'api_version' in response.json():
+        api_version = response.json()['api_version']
+    else:
+        api_version = None
+    return recommends, api_version
+
+
+def send_feedback(option_idx, latest_commands, processed_exception=None, recommends=None, accepted_recommend=None,
+                  api_version=None):
 
+    # initialize feedback data
+    # If you want to add a new property to the feedback, please initialize it here in advance and place it with ’None‘ to prevent parameter loss due to parsing errors.
```

Comment:
```suggestion
    # If you want to add a new property to the feedback, please initialize it here in advance and place it with 'None' to prevent parameter loss due to parsing errors.
```